pkg/lib/cockpit.d.ts

```diff
@@ -250,7 +250,7 @@ declare module 'cockpit' {
         read(): Promise<T>;
         replace(new_content: T | null, expected_tag?: FileTag): Promise<FileTag>;
         watch(callback: FileWatchCallback<T>, options?: { read?: boolean }): FileWatchHandle;
-        modify(callback: (data: T) => T, initial_content?: string, initial_tag?: FileTag): Promise<[T, FileTag]>;
+        modify(callback: (data: T) => T | null, initial_content?: string, initial_tag?: FileTag): Promise<[T, FileTag]>;
```

Comment:
The `data` passed to the callback can also be `null` in case the file was missing to start with.

Reply:
Ah, thanks!